lib/tasks/serialisations.rake

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+namespace :serialisations do
+  desc 'Rerun serialisations for all published content'
+  task rerun: :environment do
+    count = 0
+    Contribution.published.select do |contribution|
```

Comment:
I think this should be `Contribution.published.find_each`

Reply:
that's what I tried first, but it gives: 
```
NoMethodError: undefined method `find_each' for #<Mongoid::Criteria:0x000055b9239d66d8>
Did you mean?  find_all
```
and find_all codeclimate didn't like.